app/app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const getAllTopics = require("./app.controller");
+
+const app = express();
+
+app.get("/api/topics", getAllTopics);
+
+app.use(express.json());
```

Comment:
We don't need this middleware at this point, please remove all the code that is not related to this ticket. We can always bring it back later when we actually need it